services/order.py

```diff
@@ -0,0 +1,32 @@
+from db.models import User
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
+    user = User.objects.get(username=username)

```

Comment:
Code Style: Use `get_user_model()` instead of `User`, it is the best practice. Replace `User` with the call to `get_user_model()`.